### PR TITLE
fixed the test application compile issue: added console_print.c.

### PR DESCRIPTION
**Pull request by MiaYWei**
Makefile update by adding console_print.c.

### CODE REVIEW

**Comment by MiaYWei** [comment]
Also added return value checking when removing card from player cards list.

**Review by aditibhardwaj98 — APPROVED** [review]

File: src/computer_player.c [hunk 1]
```diff
@@ -489,8 +489,7 @@ int computer_process_request_card(void)
 {
     int result = 0;
     Card_t draw_card = draw_one_card();
-    printf("Conputer draws a new card from the draw pile \n");
-    printf("No playable card on hand, drawing a new card from deck (%s,%s).\n", CARD_COLOR_STRING[draw_card.color], CARD_NAME_STRING[draw_card.name]);//TODO Remove this line after testing
+    printf("COMPUTER requested a new card, since no playable card on hand. \n");
```
Comment (krithigasr) [inline src/computer_player.c]: "Computer draws a new card from the pile" -- > Can you change to this please?

File: src/human_player.c [hunk 2]
```diff
@@ -42,7 +42,7 @@ int request_card(PlayerType_e PlayerType)
 {
     g_card_requested = true;
     Card_t card = draw_one_card();
-    printf("The card from the draw pile is (%s,%s) \n", CARD_COLOR_STRING[card.color], CARD_NAME_STRING[card.name]);
+    printf("The requested card assigned to HUMAN is (%s,%s) \n", CARD_COLOR_STRING[card.color], CARD_NAME_STRING[card.name]);
```
Comment (krithigasr) [inline src/human_player.c]: I think the old one sounds better..

**Review by krithigasr — APPROVED** [review]

File: src/computer_player.c (hunk 1 above)
Comment (MiaYWei) [reply src/computer_player.c]: done

File: src/human_player.c (hunk 2 above)
Comment (MiaYWei) [reply src/human_player.c]: done

**Review by HongBo713 — APPROVED** [review]

**Pull request merged**